xsd/netex-nl.xsd

```diff
@@ -344,14 +393,23 @@
 							</xsd:sequence>
 						</xsd:complexType>
 					</xsd:element>
+					<!-- TODO: serviceLinks toegevoegd, kan dat ook anders? NeTEx-NL kent al TimingLinks -->
```

Comment:
Het probleem is voornamelijk hoe ziet een gemixte vorm er uit in een ServiceJourneyPattern.